### ISSUE TITLE
What's the lightest/simplest HTTP server for Go?

### ISSUE THREAD

**Issue by aashish-b** [Team-Triage/triage#22]
To establish the initial connection from the consumer instance to Triage.

**Comment by aryan-binazir** [comment]
Example of `net/http` you mentioned, Aashish. 

https://gobyexample.com/http-servers

**Comment by aashish-b** [comment]
> Example of `net/http` you mentioned, Aashish.
> 
> https://gobyexample.com/http-servers

Yeah this link and a short video tutorial are my resources for implementing this; seems good.
